frontend/src/lib/utils/eventUsageLogic.ts

```diff
@@ -370,18 +375,24 @@ export const eventUsageLogic = kea<
             }
 
             properties.total_event_actions_count = (properties.events_count || 0) + (properties.actions_count || 0)
+            // If we're aggregating this query by groups
+            properties.aggregating_by_groups = filters.aggregation_group_type_index != undefined
+            // If groups are being used in this query
+            properties.using_groups =
+                hasGroupProperties(filters.properties) || filters.breakdown_group_type_index != undefined
 
             let totalEventActionFilters = 0
-            filters.events?.forEach((event) => {
-                if (event.properties?.length) {
-                    totalEventActionFilters += event.properties.length
+            const entities = (filters.events || []).concat(filters.actions || [])
+            entities.forEach((entity) => {
+                if (entity.properties?.length) {
+                    totalEventActionFilters += entity.properties.length
+                    properties.using_groups = properties.using_groups || hasGroupProperties(entity.properties)
                 }
-            })
-            filters.actions?.forEach((action) => {
-                if (action.properties?.length) {
-                    totalEventActionFilters += action.properties.length
+                if (entity.math_group_type_index != undefined) {
+                    properties.aggregating_by_groups = true
                 }
             })
+            properties.using_groups = properties.using_groups || properties.aggregating_by_groups
```

Comment:
Not sure if we're already tracking this in a different property, but it would be quite useful to track the type of group usage (aggregation, breakdown, ...) happening (aside from this KPI).

Reply:
If we have the filter object, pretty sure we can reconstruct from there.

Until then, it's really hard to foresee important questions ahead of time - will solve for ones you know about so can you post them here?

Reply:
I don't think we're sending the entire `filters` object anymore to preserve user's privacy. I don't think we need a lot here, just knowing if you're breaking down by groups or doing unique bys seems like a solid start.